### PR TITLE
Milestone 1: Setup and Controllers

### PR DESCRIPTION
**Pull request by jaflih**
### Milestone 1: Setup and Controllers

- Created a new app. 
- Initialized the project with Git.
- Linked project to Postgres database.
- Added URLs : 
  - Root : http://127.0.0.1:3000/ 
  - http://127.0.0.1:3000/users
  - http://127.0.0.1:3000/users/id/
  - http://127.0.0.1:3000/users/id/posts/
  - http://127.0.0.1:3000/users/id/posts/id
- Created for each URL:
  - A route.
  - An action in the correct controller.
  - A view file.
- Added placeholders for each View files 
Capture d’écran 2022-04-26 à 07 47 33 Capture d’écran 2022-04-26 à 07 47 27Capture d’écran 2022-04-26 à 07 46 46 Capture d’écran 2022-04-26 à 07 45 47

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get '/users', to: 'users#index', as: 'users'
+  get '/users/:id' , to: 'users#show', as: 'user'
+
+  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
+
```
Comment (peterrobert) [inline config/routes.rb:13]: - [ ] There is a better and a more cleaner way for defining your routes using the resources method given to us by rails. Imagine if your app is bigger with more than 20 pages, defining routes this way will become very tedious and unmaintainable. Kindly improve your routes by using the resources method. To read more about this you can follow this [article](https://guides.rubyonrails.org/routing.html#configuring-the-rails-router). For Example: 
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

**Review by peterrobert — CHANGES_REQUESTED** [review]
Hi @jaflih, 

Very nice project. I love the way you closely followed the requirements for this milestone 👏🏿 . However, there are a few changes to be made kindly check below ⬇️

### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (jaflih) [reply config/routes.rb]: @peterrobert 
ok, thank you!

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @jaflih , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaflih** [comment]
@VICTOR-CHUKWU 
thanks

**Pull request merged**